tests/test_trtllm_gen_fused_moe.py

```diff
@@ -1631,6 +1661,7 @@ def run_moe_reference_dsfp8(args):
         gemm2_weights_dequant,
         args.permute_info,
         args.use_routing_scales_on_input,
+        0,  # gated_act_type
```

Comment:
```suggestion
        GatedActType.SwiGlu.value,  # gated_act_type
```
will be more intuitive